Cargo-recent.lock

```diff
@@ -105,7 +111,7 @@ version = "0.13.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1930a4dabfebb8d7d9992db18ebe3ae2876f0a305fab206fd168df931ede293b"
 dependencies = [
- "bitcoin-internals",
+ "bitcoin-internals 0.2.0",
```

Comment:
Is this supposed to 0.2.0?

Reply:
Yeah, until we release a new version of `hashes` we will have two versions of `internals` in play.

Reply:
Its when we release the new version of secp that the duplicate goes away (indirectly because of current `secp` depending on `hashes v0.13.0` depending on `internals v0.2.0`).

Reply:
I miss-spoke, when we _depend_ on the newly released version of secp.